fmriprep/workflows/bold/confounds.py

```diff
@@ -509,7 +509,7 @@ def init_carpetplot_wf(mem_gb, metadata, cifti_output, name="bold_carpet_wf"):
             ('framewise_displacement', 'mm', 'FD')]),
         name='conf_plot', mem_gb=mem_gb)
     ds_report_bold_conf = pe.Node(
-        DerivativesDataSink(desc='carpetplot', keep_dtype=True),
+        DerivativesDataSink(desc='carpetplot', datatype="figures", extension="svg"),
```

Comment:
For some reason, the name of the intermediate carpetplot generated for surfaces has the form `.carpetplot.svg`, and that "carpetplot" is interpreted as extension.